src/pages/project/project_wizard/ProjectForm.tsx

```diff
@@ -28,9 +28,11 @@ import { ProjectFormStepper } from './ProjectFormStepper';
 interface ProjectFormProps {
   project: SaveCristinProject | CristinProject;
   suggestedProjectManager?: string;
+  toggleModal?: () => void;
+  onProjectCreated?: (value: CristinProject | ResearchProject) => void;
```

Comment:
ser ut som dette bare kan være `CristinProject`?
```suggestion
  onProjectCreated?: (value: CristinProject) => void;
```